### ISSUE TITLE
Add a test suite

### ISSUE THREAD

**Issue by maxxxxxdlp** [maxxxxxdlp/eecs-448-pixelland#45]
#### Test Suite (20%)
- [ ] Must add a one-click test option in your final product
- [ ] Prints to screen, console, or file the summary of test suite being ran and the results of those tests

```
You should run a suite of tests to verify the core functionality
You are not required to test API, libraries, or other code you didn't write
Recall, writing every possible test isn't feasible.
Write smart tests for the essential functionality of your project
```

**Comment by maxxxxxdlp** [comment]
Added a map for testing map generation: http://localhost:8080/?development&map=test

**Comment by maxxxxxdlp** [comment]
Fixed in 1fcfa06 and 19fef3a
